src/system/CategoriesModule/Twig/Extension/CategoriesExtension.php

```diff
@@ -13,13 +13,24 @@
 
 use CategoryUtil;
 use DataUtil;
-use ZLanguage;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
 
 class CategoriesExtension extends \Twig_Extension
 {
-    public function getName()
+    /**
+     * @var Request
+     */
+    private $request;
+
+    /**
+     * CategoriesExtension constructor.
+     *
+     * @param RequestStack $requestStack
+     */
+    public function __construct(RequestStack $requestStack)
     {
-        return 'zikula_categories_module';
+        $this->request = $requestStack->getMasterRequest();
```

Comment:
good practice is to delay getting the request from the stack until you need the value.